demo/fullstack-app/src/pages/api/createPoints.ts

```diff
@@ -0,0 +1,74 @@
+import { type NextApiRequest, type NextApiResponse } from 'next'
+import { PointsWriter } from '@ceramic-solutions/points'
+import type { ModelInstanceDocument } from '@composedb/types'
+import { PointsContent, type AggregationContent } from '@/utils/types'
+import { fromString } from 'uint8arrays'
+
+const CERAMIC_PRIVATE_KEY: string = process.env.CERAMIC_PRIVATE_KEY ?? ''
+const aggregationModelID: string | undefined = process.env.AGGREGATION_ID ?? undefined
+
+interface Request extends NextApiRequest {
+  body: {
+    recipient: string
+    amount: number
+    context: string
+  }
+}
+
+interface Response extends NextApiResponse {
+  status(code: number): Response
+  send(data: { contextTotal: number; total: number } | { error: string }): void
+}
+
+export default async function handler(req: Request, res: Response) {
+  try {
+    const { recipient, amount, context } = req.body
+    //eslint-disable-next-line
+    const seed = fromString(CERAMIC_PRIVATE_KEY, "base16") as Uint8Array;
+
+    // create a context writer
+    const contextWriter = await PointsWriter.fromSeed({
+      aggregationModelID,
+      seed,
+    })
+
+    // create a total writer
+    const writer = await PointsWriter.fromSeed({
+      seed,
+    })
```

Comment:
You could probably initialize these outside of the request handler so they can be reused rather than re-created for every request, no?

Reply:
Definitely - amended in commit [ac67b90](https://github.com/ceramicstudio/solutions-sdk/pull/18/commits/ac67b907bdd9ed78c7a476a33a8f5630a001b19b)!